modules/services/kdeconnect.nix

```diff
@@ -0,0 +1,42 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.kdeconnect;
+  package = pkgs.kdeconnect;
+
+in
+
+{
+  meta.maintainers = [ maintainers.adisbladis ];
+
+  options = {
+    services.kdeconnect = {
+      enable = mkEnableOption "Kdeconnect";
```

Comment:
Perhaps `mkEnableOption "KDE Connect"` to better match what it is [called upstream](https://community.kde.org/KDEConnect)?